app/stories/userpages.stories.jsx

```diff
@@ -0,0 +1,51 @@
+// $flow
```

Comment:
Any chance we could get the new select control exposed here?

Reply:
Added now.